sklearn/metrics/scorer.py

```diff
@@ -290,7 +307,7 @@ def make_scorer(score_func, greater_is_better=True, needs_proba=False,
     Examples
     --------
     >>> from sklearn.metrics import fbeta_score, make_scorer
-    >>> ftwo_scorer = make_scorer(fbeta_score, beta=2)
+    >>> ftwo_scorer = make_scorer(fbeta_score, name='fbeta_score', beta=2)
```

Comment:
Why should we use the magic of `name` when we actually mean an accessible
property?

On 26 May 2016 at 12:48, Henry Lin <notifications@github.com> wrote:

> In sklearn/metrics/scorer.py
> <https://github.com/scikit-learn/scikit-learn/pull/6697#discussion_r64685126>
> :
>
> > @@ -290,7 +307,7 @@ def make_scorer(score_func, greater_is_better=True, needs_proba=False,
> >      Examples
> >      --------
> >      >>> from sklearn.metrics import fbeta_score, make_scorer
> > -    >>> ftwo_scorer = make_scorer(fbeta_score, beta=2)
> > +    >>> ftwo_scorer = make_scorer(fbeta_score, name='fbeta_score', beta=2)
>
> Aside: the name parameter to make_scorer seems clunky to me. Is there any
> interest in the future to make sure that all scikit-learn scorers have a
> __name__ attribute?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/scikit-learn/scikit-learn/pull/6697/files/5339e1f5ad2c37c2dc51fd06681abc06eeda8148#r64685126>
>